pkg/process/logging.go

```diff
@@ -29,7 +33,7 @@ var (
 	logDev      = flag.Bool("log.development", isDev(), "if true, set logging to development mode")
 	logCaller   = flag.Bool("log.caller", isDev(), "if true, log function filename and line number")
 	logStack    = flag.Bool("log.stack", isDev(), "if true, log stack traces")
-	logEncoding = flag.String("log.encoding", "console", "configures log encoding. can either be 'console' or 'json'")
+	logEncoding = flag.String("log.encoding", "pretty", "configures log encoding. can either be 'console', 'pretty', or 'json'")
```

Comment:
Do we need to notify the community in case they have built tooling around the `json` output?

Reply:
I dont think so, as the pure json output is still available

Reply:
I mean just notifying or including in release notes that "When you are relying on JSON log output from the storage node please modify setting XYZ".

Reply:
Yeah, thats something worth mentioning to @littleskunk to keep track of 👍